vars/starter_dev.yml

```diff
@@ -0,0 +1,32 @@
+---
+drupal_build_composer_project: false
+drupal_deploy: true
+#drupal_deploy_repo: https://github.com/Islandora/islandora-starter-site.git
+#drupal_deploy_version: main
+drupal_deploy_repo: https://github.com/adam-vessey/islandora-starter-site.git
+drupal_deploy_version: feature/playbook-standard-replacement
```

Comment:
Back to needing to be pointed back upstream after https://github.com/Islandora/islandora-starter-site/pull/6 is merged.

Reply:
Addressed in https://github.com/Islandora-Devops/islandora-playbook/pull/226/commits/b3f1bca7b8f8dfe18f16b10e60e259e3a984395e